docs/dev/makefile.rst

```diff
@@ -36,42 +36,61 @@ Python environment
 
    ``source ./local/py3/bin/activate``
 
-With Makefile we do no longer need to build up the virtualenv manually (as
-described in the :ref:`devquickstart` guide).  Jump into your git working tree
-and release a ``make pyenv``:
+With Makefile we do no longer need to build up the virtualenv manually. Jump
+into your git working tree and release a ``make pyenv``:
 
 .. code:: sh
 
    $ cd ~/searx-clone
    $ make pyenv
-   PYENV     usage: source ./local/py3/bin/activate
+   PYENV     [virtualenv] usage: source ./local/py3/bin/activate
+   PYENV     [virtualenv] installing requirements*.txt into ./local/py3
    ...
+   Successfully installed ...
 
 With target ``pyenv`` a development environment (aka virtualenv) was build up in
 ``./local/py3/``.  To make a *developer install* of searx (:origin:`setup.py`)
 into this environment, use make target ``install``:
 
-.. code:: sh
+.. code:: txt
 
    $ make install
-   PYENV     usage: source ./local/py3/bin/activate
-   PYENV     using virtualenv from ./local/py3
-   PYENV     install .
+   PYENV     [virtualenv] usage: source ./local/py3/bin/activate
+   PYENV     [virtualenv] using ./local/py3 // glob pattern requirements*.txt --> sha256 OK
+   ...
+   ModuleNotFoundError: No module named 'searx'
+   PYENV     [pyenvinstall]  pip install -e .\[test\]
+   ...
+     Running setup.py develop for searx
+   Successfully installed ... searx ...
 
 You have never to think about intermediate targets like ``pyenv`` or
 ``install``, the ``Makefile`` chains them as requisites.  Just run your main
 target.
 
 .. sidebar:: drop environment
 
-   To get rid of the existing environment before re-build use :ref:`clean target
-   <make clean>` first.
+   To get rid of the existing environment before re-build use: :ref:`make clean`
 
-If you think, something goes wrong with your ./local environment or you change
-the :origin:`setup.py` file (or the requirements listed in
+If you think, something goes wrong with your ``./local`` environment or you
+change the :origin:`setup.py` file (or the requirements listed in
 :origin:`requirements-dev.txt` and :origin:`requirements.txt`), you have to call
 :ref:`make clean`.
 
+``PYENV_REQ``: requirement files (glob pattern: ``requirements*.txt``)
+  For ever reasons, if you want to install only ``requirements.txt``
+
+  .. code:: txt
```

Comment:
Same: Why is it `txt`?

Reply:
see https://github.com/searx/searx/pull/2524#discussion_r579314307